Pipfile

```diff
@@ -20,3 +21,6 @@ pydocstyle = "*"
 isort = "*"
 pylint = "*"
 pytest = "*"
+pytest-mypy = "*"
+mypy = "*"
+typing-extensions = "*"
```

Comment:
Do we need this extension? it seems that we use Python3.7 for this project and the documentation says that maybe we can skip it: https://pypi.org/project/typing-extensions/

Reply:
Yes, we do. It's really weird but tests are failing without it (due to `mypy` dependencies, I believe). :/

Reply:
yes, you are right :) https://github.com/python/mypy/blob/master/setup.py

![Screen Shot 2019-11-08 at 12 05 10 PM](https://user-images.githubusercontent.com/18484872/68472099-0be53080-0220-11ea-9f63-59ce855f4463.png)